src/core/UI/RMXOverlayViewController.m

```diff
@@ -248,20 +274,57 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
   return [[self cellClassForVariable:variable] cellHeight];
 }
 
-#pragma mark - RMXOverlayViewDelegate
+#pragma mark - RMXCellDelegate
+
+- (void)cellRequestedFullScreenOverlay:(RMXCell *)cell {
+  [self maximizePanel];
+}
 
-- (void)touchStartedAtPoint:(CGPoint)point withEvent:(UIEvent *)event {
-  // No-op.
+#pragma mark - RMXShareLinkCellDelegate
+
+- (void)linkButtonWasTapped:(UIButton *)linkButton {
```

Comment:
The cell with the link needs to be revisited but this PR is already pretty long so I'd rather get it in and then make changes to it in a separate PR.